staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1243,7 +1243,10 @@ type ManagedFieldsEntry struct {
 	// APIVersion field. It is necessary to track the version of a field
 	// set because it cannot be automatically converted.
 	APIVersion string `json:"apiVersion,omitempty" protobuf:"bytes,3,opt,name=apiVersion"`
-	// Time is timestamp of when these fields were set. It should always be empty if Operation is 'Apply'
+	// Time is the timestamp of when the ManagedFields entry was added. The
+	// timestamp will also be updated if a field is added, the manager
+	// changes any of the owned fields value or removes a field. The
+	// timestamp does not update when a field is stolen from the entry.
```

Comment:
maybe explain stolen?
```suggestion
	// timestamp does not update when a field is removed from the entry
	// because another manager took it over.
```